projects/ad4134_fmc/common/ad4134_bd.tcl

```diff
@@ -0,0 +1,81 @@
+create_bd_intf_port -mode Master -vlnv analog.com:interface:spi_master_rtl:1.0 ad4134_di
+create_bd_port -dir O ad4134_odr
+
+# create a SPI Engine architecture for ADC
+
+source $ad_hdl_dir/library/spi_engine/scripts/spi_engine.tcl
```

Comment:
use this as reference 
https://github.com/analogdevicesinc/hdl/blob/fix_spi_engine_script/projects/pulsar_adc_pmdz/common/pulsar_adc_pmdz_bd.tcl#L3-L15